server/src/pages/shotindex/view.js

```diff
@@ -91,6 +95,13 @@ class Body extends React.Component {
     );
   }
 
+  getShotType(dimensions) {
```

Comment:
I'd call this getClipType, since the shot is the entire thing, and this is about the clip image.